test/empty_suite/verify_empty.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# Verifies that argv[1] is an empty file.
+
+set -eux
+
+test -f $1
```

Comment:
If you apply my above suggestion this would be rewritten to `test -f $(rlocation $1)`.

Essentially, when `--noenable_runfiles` Bazel writes a file `MANIFEST` contains a mapping of logical path to on disk path for every file in the `data` attribute. the `rlocation` function parses the file given the logical path and returns the absolute path. If runfiles are enabled you don't need this because then bazel will create a symlink for logical path -> on disk path.